src/DIPS.Xamarin.UI/Controls/Popup/PopupBehavior.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Extensions;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Popup
+{
+    /// <summary>
+    /// Behavior to be added to an item you want to open a popup from. This item _has_ to be inside a PopupLayout somehow. Else, nothing would happend.
+    /// </summary>
+    [ContentProperty(nameof(Content))]
+    public class PopupBehavior : Behavior<View>
+    {
+        private readonly Command m_onTappedCommand;
+        private View? m_attachedTo;
+        /// <summary>
+        /// Creates the instance
+        /// </summary>
+        public PopupBehavior()
+        {
+            m_onTappedCommand = new Command(ShowPopup);
+        }
+
+        /// <summary>
+        /// Handels attaching to item
+        /// </summary>
+        /// <param name="bindable"></param>
+        protected override void OnAttachedTo(View bindable)
+        {
+            m_attachedTo = bindable;
+            if (bindable is Button button)
+            {
+                button.Command = m_onTappedCommand;
+            }
+            else
+            {
+                bindable.GestureRecognizers.Add(new TapGestureRecognizer { Command = m_onTappedCommand });
+            }
+
+            base.OnAttachedTo(bindable);
+        }
+
+        /// <summary>
+        /// Handles detaching to item
+        /// </summary>
+        /// <param name="bindable"></param>
+        protected override void OnDetachingFrom(View bindable)
+        {
+            base.OnDetachingFrom(bindable);
+        }
+
+        /// <summary>
+        /// <see cref="BindingContextFactory" />
+        /// </summary>
+        public static readonly BindableProperty BindingContextFactoryProperty =
+            BindableProperty.Create(nameof(BindingContextFactory), typeof(Func<object>), typeof(PopupBehavior), null);
+
+        /// <summary>
+        /// A func which should return the requested viewModel for the popup when it opens.
+        /// Can be used to create a new instance of a filter, and have the save command send it back with CommandParameter="{Binding}".
```

Comment:
No need to write this in the xml doc, we can write example usage in wiki regarding this.

Reply:
Add that the parent `BindingContext` is set if this is left null. Double check that it does also if you havent.